dockerfiles/theia/Dockerfile

```diff
@@ -10,7 +10,7 @@
 ###
 # Builder Image
 #
-FROM #{INCLUDE:docker/${BUILD_IMAGE_TARGET}/builder-from.dockerfile}
+FROM quay.io/eclipse/che-theia-dev:next as builder
```

Comment:
why are you changing this? this will likely break downstream builds of CRW theia.

Reply:
Sorry, it was a leftover